eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/configuration/VariantProcessorConfiguration.java

```diff
@@ -66,4 +68,14 @@ VariantProcessor variantProcessor(InputParameters inputParameters) {
     ExcludeStructuralVariantsProcessor excludeStructuralVariantsProcessor() {
         return new ExcludeStructuralVariantsProcessor();
     }
+
+    @Bean
+    ContigReplacerProcessor contigReplacerProcessor(ContigMapping contigMapping) {
```

Comment:
you are not using this processor in the `compositeProcessor.setDelegates(...)` call